tools/linter/golangci-lint/.golangci.yml

```diff
@@ -37,6 +38,32 @@ linters-settings:
             desc: "use k8s.io/utils/ptr instead"
           - pkg: github.com/tetratelabs/multierror
             desc: "use errors instead"
+  importas:
+    # Do not allow unaliased imports of aliased packages.
+    no-unaliased: true
+    # Do not allow non-required aliases.
+    no-extra-aliases: false
+    alias:
+      # gateway-api
+      - pkg: sigs.k8s.io/gateway-api/apis/v1
+        alias: gwapiv1
+      - pkg: sigs.k8s.io/gateway-api/apis/v1alpha2
+        alias: gwapiv1a2
+      - pkg: sigs.k8s.io/gateway-api/apis/v1alpha3
+        alias: gwapiv1a3
+      - pkg: sigs.k8s.io/gateway-api/apis/v1beta1
+        alias: gwapiv1b1
+      - pkg: github.com/envoyproxy/gateway/api/v1alpha1
+        alias: egv1a1
+      # kubernetes api
+      - pkg: k8s.io/apimachinery/pkg/apis/meta/v1
+        alias: metav1
+      - pkg: k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1
+        alias: apiextensionsv1
+      - pkg: sigs.k8s.io/mcs-api/pkg/apis/v1alpha1
+        alias: mcsapiv1a1
+      - pkg: k8s.io/api/(\w+)/(v[\w\d]+)
```

Comment:
what's this extra one

Reply:
match in regexp, like matching `k8s.io/api/policy/v1`, the alias will be `policyv1`

Reply:
ah cool